packages/drive/package.json

```diff
@@ -30,13 +30,14 @@
     "!**/*.tsbuildinfo"
   ],
   "dependencies": {
-    "b4a": "^1.6.0",
     "corestore": "6.2.0",
     "hyperbee": "^2.0.1",
-    "hyperdrive": "^11.0.0-alpha.5",
+    "hyperdrive": "github:Nazeh/hyperdrive-next#keychain",
```

Comment:
Is this intended as a temp solution until we get something more official?

Reply:
Yes https://github.com/hypercore-protocol/hyperdrive-next/pull/18